commands/test.js

```diff
@@ -13,30 +13,66 @@ const {TestAppRuntime} = require('../build/test-runtime');
 exports.run = async function(
   {
     dir = '.',
-    watch,
     debug,
     match,
     env,
     testFolder,
-    updateSnapshot,
-    coverage,
     configPath,
     // Allow snapshots to be updated using `-u` as well as --updateSnapshot.
     // We don't document this argument, but since jest output automatically
     // suggests this as a valid argument, we support it in case it's used.
     u,
+    updateSnapshot,
+    // Arguments which are passed through into jest
+    ...rest
```

Comment:
IIRC, we need to also explicitly specify arguments in index.js since we now use sade

Reply:
Seems like it's only for documentation, or when we run `--help`, but we can list them there as well!